puppet/src/control_socket_client.rs

```diff
@@ -5,15 +5,15 @@ use anyhow::{bail, Context, Result};
 pub use tml_tcp_control_socket_client as tcp;
 
 // UNIX SeqPacket control socket transport implementation:
-#[cfg(feature = "transport_unix_seqpacket")]
+#[cfg(all(feature = "transport_unix_seqpacket", target_os = "linux"))]
```

Comment:
Hm, it's not great that we'd need to use conditional compilation everywhere. I propose that instead we retire the UNIX SeqPacket connector for now, and if we are ever to re-introduce it we can ensure that the underlying tokio crate also compiles on macOS.

It's a shame that the Darwin kernel doesn't have `SOCK_SEQPACKET` support.

Reply:
Sounds good to me. I'm pretty sure Darwin has _some_ manner of seqpacket support, just that the APIs are different, but figuring out how to use it would be I think too much work given that we're not actively using seqpacket anymore.